services/provider/server/server.go

```diff
@@ -191,7 +194,11 @@ func (s *OCSProviderServer) OffboardConsumer(ctx context.Context, req *pb.Offboa
 	if err != nil {
 		return nil, status.Errorf(codes.Internal, "failed to delete storageConsumer resource with the provided UUID. %v", err)
 	}
-
+	// remove noobaa account
+	err = s.consumerManager.DeleteNoobaaAccount(ctx, req.StorageConsumerUUID)
+	if err != nil {
+		return nil, status.Errorf(codes.Internal, "failed to delete noobaaAccount resource with the provided UUID. %v", err)
+	}
```

Comment:
This will fail as we are fetching via StorageConsumer via UUID inside DeleteNoobaaAccount, let's move this before the deletion of storageConsumer

Reply:
Moved above deletion of storageConsumer.